language/diem-framework/compiled/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 #![forbid(unsafe_code)]
 
 pub mod transaction_scripts;
+pub mod tmp_new_transaction_script_builders;
```

Comment:
So the idea is that this module will be removed once the `script fun` work is complete?

Reply:
Correct. This is only a temporary workaround so that we can have builders for any new scripts created between this PR and when `script fun`s are fully supported.